src/main/java/io/vertx/core/json/JsonObject.java

```diff
@@ -88,6 +88,26 @@ public JsonObject(Buffer buf) {
     }
   }
 
+  /**
+   * Create a JsonObject from some mapping pairs from arguments.
+   * Ignore the last argument if the number of arguments is not even.
+   *
+   * @param args The key/value pairs.
+   */
+  public static JsonObject of(Object ...args) {
+    JsonObject obj = new JsonObject();
+
+    int length = args.length;
+
+    for (int i = 0; i + 1 < length; i += 2) {
+      String k = (String) args[i];
```

Comment:
Perhaps instead of forcing a cast, always perform a `.toString()`? This prevents `ClassCastException` and works a bit like Jackson data format extensions. (For example, in Yaml/CBOR non string keys are converted to String and no errors are thrown)